### DISCUSSION TITLE
zopen update all - thought

### DISCUSSION THREAD

**Discussion by lbdyck** [zopencommunity/meta#821] [General]
when doing an update all should it first check for an update to `meta` and if found do it and then reinvoke using the updated `meta` ?

**Comment by lbdyck** [comment]
nudge

**Reply by IgorTodorovskiIBM** [reply]
I like this idea, we may even want to do this in all cases. I believe brew does this, it checks if there's an update to brew prior to installing a package. @DevonianTeuchter , what do you think?

**Reply by DevonianTeuchter** [reply]
Seems like a sensible idea.
